README.md

```diff
@@ -29,7 +29,7 @@ jobs:
     steps:
       - name: Trigger
         id: trigger
-        uses: adobe-rnd/github-purge-cache-action@master
+        uses: adobe-rnd/github-purge-cache-action
```

Comment:
are you sure? I tried in another workflow without the `@master` and got: 

![image](https://user-images.githubusercontent.com/917628/101589750-67d64600-3a2c-11eb-8ef1-60e7c1947309.png)

Reply:
Irks. This wasn't supposed to  be part of this PR…this is an old bug…